### PR TITLE
MSEARCH-9: Adjust mappings for multilang fields.

### PR DESCRIPTION
**Pull request by bohdan-suprun**

### CODE REVIEW

File: src/main/java/org/folio/search/service/IndexService.java [hunk 1]
```diff
@@ -56,6 +61,9 @@ public FolioCreateIndexResponse createIndex(String resourceName, String tenantId
   public FolioIndexOperationResponse updateMappings(String resourceName, String tenantId) {
     var index = getElasticsearchIndexName(resourceName, tenantId);
     var mappings = mappingHelper.getMappings(resourceName);
+
+    log.info("Updating mappings for resource [{}] for tenant [{}] - [{}]",
```
Comment (pfilippovEPAM) [inline src/main/java/org/folio/search/service/IndexService.java:65]: log format always anywhere in search should be like ```$message [$parameter: $value]```

Comment (bohdan-suprun) [reply src/main/java/org/folio/search/service/IndexService.java]: thanks, changed.

**Review by pfilippovEPAM — APPROVED** [review]

File: src/main/java/org/folio/search/service/es/SearchMappingsHelper.java [hunk 2]
```diff
@@ -28,11 +29,13 @@
 public class SearchMappingsHelper {
 
   private static final String MAPPING_PROPERTIES_FIELD = "properties";
+  private static final String MULTILANG_SOURCE_FIELD = "src";
```
Comment (hjiebsco) [inline src/main/java/org/folio/search/service/es/SearchMappingsHelper.java:32]: Line 128 still uses hardcoded `src`

File: src/test/java/org/folio/search/controller/ConfigControllerIT.java [hunk 3]
```diff
@@ -88,14 +90,15 @@ void shouldUseConfiguredLanguagesDuringMapping() {
...
-      .languages(languageCodes)
+      .languages(List.of("eng", "rus", "fre"))
       .title("This is title");
 
     kafkaTemplate.send(INVENTORY_INSTANCE_TOPIC, newInstance.getId(),
       eventBody(INSTANCE_RESOURCE, newInstance));
 
     final var indexedInstance = getIndexedInstanceById(newInstance.getId());
 
+    assertThat((Map<String, Object>) getMapValueByPath("title", indexedInstance), aMapWithSize(3));
```
Comment (hjiebsco) [inline src/test/java/org/folio/search/controller/ConfigControllerIT.java:101]: Maybe add `assertNull(getMapValueByPath("title.fre", indexedInstance));`

**Review by hjiebsco — APPROVED** [review]

File: src/main/java/org/folio/search/service/es/SearchMappingsHelper.java (hunk 2 above)
Comment (bohdan-suprun) [reply src/main/java/org/folio/search/service/es/SearchMappingsHelper.java:32]: changed.

File: src/test/java/org/folio/search/controller/ConfigControllerIT.java (hunk 3 above)
Comment (bohdan-suprun) [reply src/test/java/org/folio/search/controller/ConfigControllerIT.java:102]: added.

**Pull request merged**